apps/android/common/src/commonMain/resources/distribute/SimpleX.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Type=Application
+Name=SimpleX
+Comment=Messenger for the future
```

Comment:
where does it show? needs to be changed.

Reply:
Linux desktop environments like kde,xfce,gnome